### PR TITLE
feat(deps): support React up to 20

### LINKED ISSUE #117: Update to react 17
I don't think that description is needed.

I don't mind to do, just update me if you allow.

Thanks.

### PR DESCRIPTION
**Pull request by naorpeled**
closes #117, #123, #134

### CODE REVIEW

**Comment by naorpeled** [comment]
@alewin this should hopefully do the trick :)

**Comment by alewin** [comment]
great @naorpeled this solves a lot of issues and compatibility!
just a note: react has supported hooks since version `v16.8.0`, so since `useWorker` is a hook we can't guarantee support for the `v16.0.0`version, which doesn't support hooks

**Comment by naorpeled** [comment]
> great @naorpeled this solves a lot of issues and compatibility!
> just a note: react has supported hooks since version `v16.8.0`, so since `useWorker` is a hook we can't guarantee support for the `v16.0.0`version, which doesn't support hooks 

Great point, thanks!!!

File: packages/useWorker/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "test": "npm run build && npm run test:unit"
   },
   "peerDependencies": {
-    "react": "^16.8.0"
+    "react": ">=16.0.0 <20.0.0"
```
Comment (alewin) [inline packages/useWorker/package.json:18]:
```suggestion
    "react": ">=16.8.0 <20.0.0"
```

ref: https://legacy.reactjs.org/docs/hooks-reference.html

**Review by alewin — APPROVED** [review]

**Pull request merged**